### PR TITLE
Small bug fix for the ADLSESetup.Page.al file.

### PR DESCRIPTION
**Pull request by Bertverbeek4PS**
The ClientId and ClientSecret fields were not being set correctly.
@DuttaSoumya

### CODE REVIEW

**Comment by DuttaSoumya** [comment]
Hi @Bertverbeek4PS,

Thanks for the suggestion!

Please refer to the commit https://github.com/microsoft/bc2adls/commit/dc3aa6a0bbb9723484e93722812b162925996199 - those two lines were actually taken out on purpose :-) You see, a user could inadvertently expose the setup page as a web service and suddenly those secrets become visible- this was a security hole! Therefore it was decided at that time to never _fetch_ the values of the secrets- only _change_ or _override_ them. 

In case the change you made is inspired by the annoyance of having blanks shown up on the fields on the page, even when you know that they are actually filled in, may I propose a different way to solve this issue? 

Instead of the line,

    ClientSecret := ADLSECredentials.GetClientSecret();
we could have something like,

    ClientSecret := 'secret not shown';
This would, of course have the issue of the fields filled with "stars" even if it is blank, perhaps a better way would be to change the line to this instead,

    if ADLSECredentials.IsClientSecretSet() then
        ClientSecret := 'secret not shown';
Of course, it would necessitate creation of a few new functions on the codeunit [ADLSECredentials](https://github.com/microsoft/bc2adls/blob/main/businessCentral/src/ADLSECredentials.Codeunit.al) and perhaps a new text constant.

Best regards,
Dutta.

**Comment by Bertverbeek4PS** [comment]
Thanks @DuttaSoumya wasn't aware of that. But that would more a platform bug. Because if you put the ExtendenDatatype to masked the API should respect that.

I have made the change because there where indeed blanks on the fields. So I though I haven't filled it in.

Indeed then the text constant 'secret not shown' would be better.
I will create the change and create a commit the coming days.

Thanks!

File: businessCentral/src/ADLSECredentials.Codeunit.al [hunk 1]
```diff
@@ -103,6 +103,13 @@ codeunit 82565 "ADLSE Credentials"
         SetSecret(ClientSecretKeyNameTok, NewClientSecretValue);
     end;
 
+    [NonDebuggable]
+    internal procedure IsClientSecret(): Boolean
```
Comment (DuttaSoumya) [inline businessCentral/src/ADLSECredentials.Codeunit.al:107]: Rename to `IsClientSecretSet`?

Also Client ID should also be treated as a secret. Therefore a similar function called `IsClientIDSet`.

No need for internal keyword- the entire codeunit is internal.

File: businessCentral/src/ADLSECredentials.Codeunit.al [hunk 2]
```diff
@@ -103,6 +103,13 @@ codeunit 82565 "ADLSE Credentials"
...
+    begin
+        If GetClientSecret() <> '' then
+            exit(true);
+    end;
+
```
Comment (DuttaSoumya) [inline businessCentral/src/ADLSECredentials.Codeunit.al:112]: <minor> 
the body may be written in a short form, as 

    exit(GetClientSecret() <> '');

File: businessCentral/src/ADLSESetup.Page.al [hunk 3]
```diff
@@ -219,13 +219,18 @@ page 82560 "ADLSE Setup"
...
+    var
+        ClientSecretLbl: Label 'Secret not shown';
 
     trigger OnInit()
     begin
         Rec.GetOrCreate();
         ADLSECredentials.Init();
         StorageTenantID := ADLSECredentials.GetTenantID();
         StorageAccount := ADLSECredentials.GetStorageAccount();
+        ClientID := ADLSECredentials.GetClientID();
```
Comment (DuttaSoumya) [inline businessCentral/src/ADLSESetup.Page.al:231]: Replace by,

    if ADLSECredentials.IsClientIDSet() then 
        ClientID := ClientSecretLbl;
Please add the `()` at the end of the function call for the client secret.

**Review by DuttaSoumya — CHANGES_REQUESTED** [review]
Looks really good- this will solve the issue. A few comments added.

Please do not forget to increment the minor build number in the app.json version property.

**Review by DuttaSoumya — APPROVED** [review]

**Pull request merged**